assets/js/base/components/chip/style.scss

```diff
@@ -0,0 +1,40 @@
+.wc-block-components-chip {
+	display: inline-block;
+	background: $core-grey-light-500;
+	padding: 0.365em 0.5em;
+	margin: 0 0.365em 0.365em 0;
+	color: $core-grey-dark-800;
+	border-radius: 0;
+	line-height: 1em;
+
+	&.wc-block-components-chip--radius-small {
+		border-radius: 3px;
+	}
+	&.wc-block-components-chip--radius-medium {
+		border-radius: 0.433em;
+	}
+	&.wc-block-components-chip--radius-large {
+		border-radius: 0.865em;
+		padding: 0.365em 0.75em;
+	}
+
```

Comment:
Do we have a particular reason for defining specific radius classes? I'd hold off on this until we have a specific need, and/or just allow client code to override a nice default. Also curious to mix px/em units.

My preference would be to default to a big radius so the ends of the chip are semicircle. (Just my opinion, I defer to design / @garymurray !)

<img width="291" alt="Screen Shot 2020-02-26 at 4 20 53 PM" src="https://user-images.githubusercontent.com/4167300/75308703-f7eb3a80-58b3-11ea-83eb-a497271b017f.png">

Reply:
Gary already commented that we needed round chips for coupons. The other chips need to be partially rounded (when this is implemented for filters, which is a cooldown task). The radius options are here to support that and were suggested by @nerrad 

The mix of em/px is justified IMO. EM to ensure we round fully regardless of font size, and px to ensure "small" is actually small/only slightly rounded. There is method to my madness.